tools/pkg-config: pass arguments at the end

0x5a17ed opened the pull request:
Go for openwrt passes pkg-config arguments in the format of

        pkg-config --cflags -- pkg-name 

which in turn will be passed down to the real pkg-config as something
like

        pkg-config.real --cflags -- pkg-name --define...

and causes the real pkg-config implementation to missinterpret the given
argument list.

This also helps to fix https://github.com/golang/go/issues/27940

hauke commented:
Please add a Signed-of-by: line to the commit description, otherwise it looks good.

0x5a17ed commented:
Signed-off-by line has been added to the commit description by executing `git commit --amend -s --no-edit`.

corny commented:
@hauke ready to be merged?

0x5a17ed committed:
tools/pkg-config: pass arguments at the end

Go for openwrt passes pkg-config arguments in the format of

        pkg-config --cflags -- pkg-name

which in turn will be passed down to the real pkg-config as something
like

        pkg-config.real --cflags -- pkg-name --define...

and causes the real pkg-config implementation to missinterpret the given
argument list.

This also helps to fix https://github.com/golang/go/issues/27940

Signed-off-by: Arthur Skowronek <arthur.skowronek@tuta.io>

hauke commented:
Thank you for your pull request it was merged into master.
